### PR TITLE
deprecating the image py-ews

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9764) 
[Bump exchangelib in EWSv2 docker](https://jira-dc.paloaltonetworks.com/browse/CIAC-10403)

## Description
- updating docker to new based python debian version
- removing temporary changes that were relevant to previous exchangelib version since it was bumped.
- removing compliance commands from yml and python

### CODE REVIEW

File: Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py [hunk 1]
```diff
@@ -28,6 +28,8 @@
 from exchangelib.version import (EXCHANGE_2007, EXCHANGE_2010,
                                  EXCHANGE_2010_SP2, EXCHANGE_2013,
                                  EXCHANGE_2016, EXCHANGE_2019)
+from exchangelib.protocol import NoVerifyHTTPAdapter  # noqa: E402
```
Comment (JudahSchwartz) [inline Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py:31]: lets remove all the powershel code and commands

File: Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py [hunk 2]
```diff
@@ -415,9 +417,7 @@ def prepare_context(credentials):  # pragma: no cover
 
 def prepare():  # pragma: no cover
     if NON_SECURE:
-        BaseProtocol.HTTP_ADAPTER_CLS = exchangelibSSLAdapter
-    else:
-        BaseProtocol.HTTP_ADAPTER_CLS = requests.adapters.HTTPAdapter
+        BaseProtocol.HTTP_ADAPTER_CLS = NoVerifyHTTPAdapter
```
Comment (JudahSchwartz) [inline Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py:420]: explain this change?

File: Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_2.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "breakingChanges":true,
+  "breakingChangesNotes":"The commands: show-only-recipients, ews-o365-start-compliance-search, ews-o365-show-only-recipients, ews-o365-purge-compliance-search-results, ews-o365-get-compliance-search-purge-status, ews-o365-remove-compliance-search, ews-o365-get-compliance-search-purge-status were deleted. Please use the commands from Security And Compliance V2 instead."
```
Comment (JudahSchwartz) [inline Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_2.json:3]: if this is here we dont also need it in the RN?

File: Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py (hunk 2 above)
Comment (maimorag) [reply Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py:420]: fixing the error:
Cannot set verify_mode to CERT_NONE when check_hostname is enabled. (85)

**Review by eyalpalo — APPROVED** [review]

**Pull request closed without merging**